modules/user-access/pages/rbac-row-policy/row-policy-ebnf.adoc

```diff
@@ -0,0 +1,103 @@
+= RBAC Row Policy EBNF
+
+== EBNF for a Package
+
+[source,gsql]
+----
+createPackage := CREATE PACKAGE packageName ["." packageName]*
+showPackage := SHOW PACKAGE [ "-r" stringLiteral
+            | [ packageName ["." packageName "."]* [ "." functionName | "*" ]]?
+
+dropPackage := DROP PACKAGE packageName ["." packageName]*
+----
+
+== EBNF overview of the GSQL functions.
+[source,gsql]
+----
+createFunction := CREATE [OR REPLACE] FUNCTION packageName
+                [ "." packageName ]* "." functionName
+                "(" [parameterList] ")"
+                RETURNS "("  baseType ")"
+                "{" functionBody "}"
+
+functionBody := [declStmt | declExceptStmt | functionBodyStmt]+
+declStmt := baseDeclStmt | accumDeclStmt
+functionBodyStmt := assignStmt            // Assignment
+                | vSetVarDeclStmt         // Declaration
+                | gAccumAssignStmt        // Assignment
+                | gAccumAccumStmt         // Assignment
+                | funcCallStmt            // Function Call
+                | functionBodyCaseStmt    // Control Flow
+                | functionBodyIfStmt      // Control Flow
+                | returnStmt              // Output
+                | raiseStmt               // Exception
+
+installFunction := INSTALL FUNCTION [installOptions]
+                [[ packageName [ "." packageName "."]*
+                [ "." functionName | "*" | "**"] | "**"| "ALL"]
+
+showFunction := SHOW FUNCTION [ * | "-r" stringLiteral
+            |[ packageName [ "." packageName "."]*
+            [ "." functionName | "*" ] ]
+
+dropFunction := DROP FUNCTION packageName
+            ["." packageName "."]* [ "." functionName | "*" ]
+----
+
+== EBNF for the object-based privileges:
```

Comment:
Please include this EBNF list to the role based access control page as well.
https://docs.tigergraph.com/tigergraph-server/current/user-access/access-control-model

Reply:
Thank you. I included a link a the top of the page. May revisit it later to added more content.